README.md

```diff
@@ -39,9 +39,16 @@ The following configuration options are available:
 | CREDENTIALS                | A map of hawk id-keys.                                                                   | -                 |
 | DATABASE\_MAX\_OPEN\_CONNS | The maximum amount of PostgreSQL database connections tigerblood will open               | 80                |
 | BIND\_ADDR                 | The host and port tigerblood will listen on for HTTP requests                            | 127.0.0.1:8080    |
-| STATSD\_ADDR               | The host and port for statsd                                                             | 127.0.0.1:8125    |
 | DSN                        | The PostgreSQL data source name. Mandatory.                                              | -                 |
 | HAWK                       | true to enable Hawk authentication. If true is provided, credentials must be non-empty   | false             |
+| VIOLATION_PENALTIES        | A map of violation names to their reputation penalty weight 0 to 100 inclusive.          | -                 |
```

Comment:
Is that a yaml map? And if so, why does it need to be converted from `map[string]string` to `map[string]uint` ?